[ADD][8.0] account_payment_order_sequence

andhit-r opened the pull request:
# 
# Payment Order Sequence

This module add feature to create payment order reference
based on payment mode. Each payment mode could have different
sequence.
# Configuration

To configure specific sequence for payment mode, you need to:
1. Open or create payment mode
2. Choose sequence to be used

Payment mode with empty sequence will use original payment
order sequence

pedrobaeza commented on account_payment_order_sequence/README.rst:
```diff
@@ -0,0 +1,78 @@
+Payment Order Sequence
+======================
+
+This module add feature to create payment order reference
```
s/add/adds

pedrobaeza commented on account_payment_order_sequence/README.rst:
```diff
@@ -0,0 +1,78 @@
+2. Choose sequence to be used
+
+Payment mode with empty sequence will use original payment
+order sequence
```
dot at the end

pedrobaeza commented on account_payment_order_sequence/README.rst:
```diff
@@ -0,0 +1,78 @@
+order sequence
+
+Usage
+=====
```
Add simple usage instructions.

andhit-r commented:
@pedrobaeza thanks for your reviews. I addressed your comments on https://github.com/OCA/account-payment/pull/104/commits/6cbda3eb46cf24aa08898e658b4244894ec1a971.

andhit-r commented:
Hi @OCA/accounting-maintainers would you kindly review this PR. Many thanks

andhit-r commented:
@mikevhe18 @azmimr67 please review

azmimr67 commented:
Functionality has been tested on runboard 👍

andhit-r commented:
Thanks @pedrobaeza
